### PR TITLE
Increased coverage (converge list)

### PR DESCRIPTION
**Pull request by retr0h**

### CODE REVIEW

File: test/unit/command/test_status.py [hunk 1]
```diff
@@ -24,12 +24,10 @@
 
 
 def test_execute(capsys, patched_main, molecule_instance):
-    molecule_instance.args = {'porcelain': False}
-
     s = status.Status({}, {}, molecule_instance)
     result = s.execute()
 
-    out, err = capsys.readouterr()
+    out, _ = capsys.readouterr()
```
Comment (alop) [inline test/unit/command/test_status.py]: what's '_' ?

Comment (alop) [inline test/unit/command/test_status.py]: What's '_' ?

Comment (retr0h) [inline test/unit/command/test_status.py]: Means the variable isn't used by anything.  It's a convention.

**Comment by retr0h** [comment]
```
  syntax: commands succeeded
  py27-ansible19-unit: commands succeeded
  py27-ansible19-functional: commands succeeded
  py27-ansible20-unit: commands succeeded
  py27-ansible20-functional: commands succeeded
  py27-ansible21-unit: commands succeeded
  py27-ansible21-functional: commands succeeded
  doc: commands succeeded
```

**Comment by signed8bit** [comment]
👍

File: test/unit/command/test_status.py (hunk 1 above)
Comment (alop) [inline test/unit/command/test_status.py]: interesting, so, you're saying `capsys.readouterr()` is going to return two things, but I only care about the first one, right?

**Pull request merged**